samples/get_portgroup.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Written by Chris Arceneaux
+# GitHub: https://github.com/carceneaux
+# Email: carcenea@gmail.com
+# Website: http://arsano.ninja
+#
+# Note: Example code For testing purposes only
+#
+# This code has been released under the terms of the Apache-2.0 license
+# http://opensource.org/licenses/Apache-2.0
+
+"""
+Python program for retrieving a port group for both VSS and DVS
+"""
+
+import atexit
+
+from tools import cli
+from pyVim import connect
+from pyVmomi import vmodl
+from pyVmomi import vim
+
+
+def get_args():
+    """
+    Adds additional args for retrieving a port group
+
+    -pg portgroup
+    """
+    parser = cli.build_arg_parser()
+
+    # because -p is reserved for 'password', we use -pg for port group name
+    parser.add_argument('-pg', '--portgroup',
```

Comment:
As you said -p is reserved for password. I think short argument can be skipped. Just use --portgroupname

Reply:
Changes made and commited.

Reply:
Hi @carceneaux , please can you check if you committed changed file? I dont see the changes